### ISSUE TITLE
2D Inverse Kinematics only work with the "Make Custom Bones" tool, not Bone2D nodes

### ISSUE THREAD

**Issue by NathanLovato** [godotengine/godot#25534]
"Make IK Chain" doesn't work with Bone2D nodes. Even if you have a chain of Bone2Ds, you must first select them, go to the bones menu in the toolbar and click "Make Custom Bones From Nodes".

Instead, I would expect to:

1. Select a chain of Bone2D nodes
2. Open the bones menu in the toolbar, click "Make IK chain"

And get an IK chain.

**Comment by Xrayez** [comment]
It's either not documented enough or the feature is really missing. Having to use either the tool or the nodes in a different manner for different purposes is confusing.

**Comment by NathanLovato** [comment]
The feature is documented, it's just that's how it worked since Godot 2 and it's not been unified with the changes in the anim tools in Godot 3.1 yet

**Comment by clayheaton** [comment]
IMO, this is a bug. You _can_ "Make IK Chain" with Bone2D nodes -- and they visually show the chain was created, with the root being yellow. It simply doesn't work. Dang, I just wasted a bunch of time thinking I had done something wrong. Glad I checked here.

**Comment by AleryBerry** [comment]
Is this solved? Still got the same issue in 2022.

**Comment by Calinou** [comment]
> Is this solved? Still got the same issue in 2022.

By definition, if you still encounter this issue, it's not resolved yet :slightly_smiling_face: 

Since 4.0 is entering feature freeze soon, this probably won't be addressed for 4.0 unless a motivated contributor does the required work.

**Comment by RobTF** [comment]
What is the deal with this? - I'm trying to get 2D skeleton animation working but nothing in 4.0 seems to match up with the latest version of the tutorial here (https://docs.godotengine.org/en/latest/tutorials/animation/cutout_animation.html#doc-cutout-animation). For one, the tutorial makes no mention of a `Skeleton2D` or `Bone2D` object, yet at least in 4.0 Beta 4 it seems to want to generate these. After that, I'm not even getting the option to make an IK chain.

Am I missing something or is this 2D skeleton feature simply not working/available?

**Comment by Zireael07** [comment]
Most likely docs haven't been updated to match 4.0 yet

**Comment by RobTF** [comment]
Yes, I had a bit more of a play and it looks like 4.0 uses a concept of skeleton modifications? I messed about and got something resembling IK although somewhat messed up, which is likely just what I was doing. I'll keep hunting for details, but if there are any useful links anyone can share that would be really appreciated!

**Comment by Joey-Einerhand** [comment]
""Make IK Chain" editor button was removed in 4.0, so this is not applicable to recent versions
